frontend/src/metabase/common/components/FilterPicker/BooleanFilterPicker.tsx

```diff
@@ -94,24 +96,27 @@ function getOptions(
   column: Lib.ColumnMetadata,
 ): Option[] {
   const operators = Lib.filterableColumnOperators(column);
-  return operators.flatMap((operator): Option[] => {
-    const operatorInfo = Lib.displayInfo(query, stageIndex, operator);
-    if (operatorInfo.shortName === "=") {
-      return [
-        { operator, name: t`True`, type: "true" },
-        { operator, name: t`False`, type: "false" },
-      ];
-    } else {
-      return [
-        {
-          operator,
-          name: operatorInfo.displayName,
-          type: operatorInfo.shortName,
-          isAdvanced: true,
-        },
-      ];
-    }
-  });
+  return operators
+    .flatMap((operator): Option[] => {
+      const operatorInfo = Lib.displayInfo(query, stageIndex, operator);
+      if (operatorInfo.shortName === "=") {
+        return [
+          { operator, name: t`True`, type: "true" },
+          { operator, name: t`False`, type: "false" },
+        ];
+      } else if (operatorsToShow.includes(operatorInfo.shortName)) {
+        return [
+          {
+            operator,
+            name: operatorInfo.displayName,
```

Comment:
one weird thing to deal with is that we want the operator to be `is-null` but we want the label to say `is-empty` - this is probably a backend change though